src/class.js

```diff
@@ -5,7 +5,17 @@
 // Add a method called `comparePasswords`.  `comparePasswords` should have a parameter
 // for a potential password that will be compared to the `password` property.
 // Return true if the potential password matches the `password` property.  Otherwise return false.
+class User {
+  constructor(options) {
+    this.email = options.email;
+    this.password = options.password;
+  }
+}
 
+User.prototype.comparePasswords = function (str) {
+  if (str === this.password) return true;
```

Comment:
This can be simplified one step further.  `return str === this.password;`  The expression is evaluated automatically to either `true` or `false` before it is returned.